src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
+  width: 200px;
+  background-color: #fff;
```

Comment:
The background-color should be defined using a variable instead of a hardcoded value. Consider using a variable like $background-color.